judo-psm-generator-jackson-api/src/main/java/hu/blackbelt/judo/psm/generator/jackson/api/OperationHelper.java

```diff
@@ -0,0 +1,188 @@
+package hu.blackbelt.judo.psm.generator.jackson.api;
+
+/*-
+ * #%L
+ * JUDO PSM Generator SDK Core API
+ * %%
+ * Copyright (C) 2018 - 2023 BlackBelt Technology
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+
+import hu.blackbelt.judo.generator.commons.StaticMethodValueResolver;
+import hu.blackbelt.judo.generator.commons.annotations.TemplateHelper;
+
+import hu.blackbelt.judo.meta.psm.PsmUtils;
+import hu.blackbelt.judo.meta.psm.namespace.Model;
+import hu.blackbelt.judo.meta.psm.namespace.NamedElement;
+import hu.blackbelt.judo.meta.psm.namespace.Namespace;
+import hu.blackbelt.judo.meta.psm.service.*;
+import hu.blackbelt.judo.meta.psm.type.Cardinality;
+
+import static hu.blackbelt.judo.psm.generator.jackson.api.JavaNamespaceHelper.*;
+import static hu.blackbelt.judo.psm.generator.jackson.api.ObjectTypeHelper.getEntity;
+import static hu.blackbelt.judo.psm.generator.jackson.api.ObjectTypeHelper.isEntity;
+
+
+@TemplateHelper
+public class OperationHelper extends StaticMethodValueResolver {
+
+    public static Boolean hasReturn(TransferOperation transferOperation) {
+        return transferOperation.getOutput() != null;
+    }
+
+    public static String operationName(TransferOperation transferOperation) {
+        return transferOperation.getName();
+    }
+
+    public static Boolean isBoundOperation(TransferOperation transferOperation) {
+        return transferOperation instanceof BoundTransferOperation;
+    }
+
+    public static Boolean isCustomOperation(TransferOperation transferOperation) {
+        return transferOperation.getImplementation() != null &&
+                transferOperation.getImplementation().isCustomImplementation();
+    }
+
+    public static Boolean hasParam(TransferOperation transferOperation) {
+        return transferOperation.getInput() != null;
```

Comment:
Same as what I mentioned for output.

Reply:
[00ec87c](https://github.com/BlackBeltTechnology/judo-psm-generator-jackson/pull/1/commits/00ec87cbc991117419a9c0480c7bbf20867b0620)